iree/hal/cuda/dynamic_symbols.c

```diff
@@ -10,16 +10,26 @@
 
 #include "iree/base/internal/dynamic_library.h"
 #include "iree/base/target_platform.h"
-#include "iree/base/tracing.h"
```

Comment:
Keep the tracing include then use 

```c
#if IREE_TRACING_FEATURES & IREE_TRACING_FEATURE_INSTRUMENTATION
```

instead of
```c
#if IREE_TRACING_FEATURES
```

matching other parts of the project that use that condition